coralproject/talk
# Fix Shortcuts help menu doesn't dismiss forever.

## What does this PR do?
Fix #1017

## How do I test this PR?
* Go to Admin
* See the shortcuts help menu in the bottom left
* Click X to dismiss forever
* Refresh page
* Menu has gone forever
